### ISSUE TITLE
ffmpeg - make dynamic graph videos

### ISSUE THREAD

**Issue by jravenel** [jupyter-naas/awesome-notebooks#272]
Following Thom Ives, Ph.D. [post on Linkedin](https://www.linkedin.com/feed/update/urn:li:activity:6875488520111312896/?commentUrn=urn%3Ali%3Acomment%3A(activity%3A6875488520111312896%2C6875494403604131840)&replyUrn=urn%3Ali%3Acomment%3A(activity%3A6875488520111312896%2C6875495625300033536)), the idea is to be able to generate dynamic graph videos with a generic notebook template. 

To be developped.

**Comment by Neelaksh-Singh** [comment]
Should this notebook emulate the same code like in the post ?

**Comment by jravenel** [comment]
@Neelaksh-Singh Yes with little changes. The idea is to be able to format it according to the naas framework (template is at the root of the repo) = we should have an input, model, and output sections + follow the naming convention "Tool_action"

**Comment by Neelaksh-Singh** [comment]
Sure I would like to work on it.

**Comment by jravenel** [comment]
@Neelaksh-Singh sure, I have added you to the naas contributor group. 

Here are the steps to contribute: 

1→ Open free account on Naas.ai 
2→ Clone awesome-notebooks repo 
3→ Change status of this Issue to "In progress" so we can know you are working on it
4→ Create new branch FFMPEG
5→ Create folder FFMPEG in awesome-notebooks folder, and adapt notebook template to the current use case (check out how Linkedin templates are done for example) 
6→ Once you are happy with the result, commit to the branch 
7→ Open a pull request and tag me as a reviewer with a little comment on what you have done, but most of the explanations should be in the notebook itself
8→ Change status of this Issue to "Review" so we can know a review is pending
9→ Link the PR to this issue for tracking in the backlog 
10→ Expect a feedback and merge in the next 48h :)

**Comment by jravenel** [comment]
@Neelaksh-Singh just assigned you to the task. Feel free to ask questions or call for help!

**Comment by Neelaksh-Singh** [comment]
Hi, I was working on this issue on the Naas manager, when my credits got expired. I would try to resolve this issue locally from my system.

**Comment by Neelaksh-Singh** [comment]
It's neelaksh48@gmail.com

**Comment by jravenel** [comment]
I added more credits to your account @Neelaksh-Singh, should be update in 15min.

**Comment by jravenel** [comment]
Hey @Neelaksh-Singh I did not manage to use it on my Naas.ai account, the apt-get sudo command don't work,  and it's normal. 
But I'm also wondering if all of this is necessary. 

Did you try youself?
Can we make a review and peer-programming session (15min) to sort this out? 
Here is my calendly https://calendly.com/jeremyravenel

**Comment by Neelaksh-Singh** [comment]
Hi, since I am out of town till the New Year, I won't be able to schedule a meeting.


> But I'm also wondering if all of this is necessary

As I was trying to emulate the same code as in the post, these libraries were required for our dataset generation. In the notebook, under the variable section, I have mentioned that if the user already has their dataset, then they may import it.
Maybe for clarity, I can put a comment in the <b>Installing Packages</b> so that only the ones who are using the notebook for testing purposes should install the above packages.

> Did you try youself?

I did try running on the Colab notebook, where it worked without any errors. When I tried running it in the Naas manager, it asked me for an FTP password (maybe for using sudo).  
You may access the Colab file [here](https://colab.research.google.com/drive/1AeeSl9-N__20uuUN6nX62Mb5ozRO32tU?usp=sharing)

**Comment by jravenel** [comment]
@Neelaksh-Singh No problem I will try to fix and simplify the image generation part. I will update your PR and let you know.

Is that ok?

**Comment by Neelaksh-Singh** [comment]
Sure
